lib/db/database_maker.py

```diff
@@ -1,6 +1,8 @@
 import os.path
 import sqlite3
 
+from lib.db.pokemon_grabber import *
+
 
 class CreateDatabase():
```

Comment:
Considering every method in this class is static does it really need to be a class?

Reply:
I agree. It does seem a bit off doing it in classes. I will change this in a future commit.